DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/GroupDelegates/SelectDelegate.cshtml

```diff
@@ -0,0 +1,58 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.GroupDelegates
+@using Microsoft.Extensions.Configuration
+@model AddGroupDelegateViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/shared/cardWithButtons.css")" asp-append-version="true">
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/delegates/allDelegates.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = $"Add delegate to {Model.GroupName} group";
+  var routeParamsForGroupDelegates = new Dictionary<string, string> {
+    { "groupId", Model.GroupId.ToString() }
+  };
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-full">
+        <h1 id="page-heading" class="nhsuk-heading-xl nhsuk-u-margin-bottom-8">@ViewData["Title"]</h1>
+      </div>
+    </div>
+    <input id="selected-group-Id" type="hidden" asp-for="@Model.GroupId" />
+
+    <partial name="SearchablePage/Configurations/_ThreeQuarterWidthSearchAndFilter" model="Model" />
+
+    @if (Model.NoDataFound)
+    {
+      <p class="nhsuk-u-margin-top-4" role="alert">
+        <b>No delegates found.</b>
+      </p>
+    }
+    else
+    {

```

Comment:
The formatting here is different to how the formatter will set it

Reply:
Ran formatter.